setup.py

```diff
@@ -24,6 +24,7 @@
     'property-caching>=1.0.3',
     'querylist>=0.2.0',
     'six==1.9.0',
+    'python-dateutil==2.4.2',
```

Comment:
Also noticed 'mock>=1.2.0', in test_requirements - it should also be fixed to be inline with requirements.txt